website/src/pages/CommercialSupport/styles.ts

```diff
@@ -88,7 +88,8 @@ const useStyles = makeStyles((theme: Theme) => ({
     },
     [theme.breakpoints.down("xs")]: {
       boxShadow:' 0px 4px 34px 21px rgba(70, 68, 151, 0.04)',
-      borderRadius: '20px 20px 40px 0px'
+      borderRadius: '20px 20px 40px 0px',
```

Comment:
where are we using theme spacing and where we are not?
kind of confused 😕

Reply:
We are using theme spacing for margins and paddings only